### ISSUE TITLE
After installing the respeaker 6-mic array driver, the arecord -L command cannot detect the sound card

### ISSUE THREAD

**Issue by 0306qianmo** [HinTak/seeed-voicecard#37]
I used https://github.com/HinTak/seeed-voicecard and then used the arecord -L command and the sound card did not appear, as shown below

**Comment by aerosh** [comment]
Hello! I'm facing the same problem. Have you found a solution to the problem?

**Comment by 0306qianmo** [comment]
> Hello! I'm facing the same problem. Have you found a solution to the problem?

No, I'm not trying it. If you figure it out, could you let me know?

**Comment by douxiaodi** [comment]
Hello! I'm facing the same problem. Have you found a solution to the problem?
